src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -115,6 +88,8 @@ private bool WritePeToStream(MetadataWriter mdWriter, Func<Stream> getPeStream,
             var managedResourceBuffer = new MemoryStream(1024);
             var managedResourceWriter = new BinaryWriter(managedResourceBuffer);
 
+            nativePdbWriterOpt?.SetMetadataEmitter(mdWriter);
```

Comment:
OK, then my logger (#926) will have to account for that input to the PDB writer. Ugh. How do I log those callbacks?